src/sidebar/components/test/Thread-test.js

```diff
@@ -214,12 +214,13 @@ describe('Thread', () => {
       noAnnotationThread = createThread();
       noAnnotationThread.annotation = undefined;
       noAnnotationThread.visible = false;
+      noAnnotationThread.parent = true;
     });
 
     it('does not render any kind of annotation component', () => {
       const wrapper = createComponent({ thread: noAnnotationThread });
 
-      assert.isFalse(wrapper.find('Annotation').exists());
+      assert.equal(wrapper.find('Annotation').prop('visibility'), 'hidden');
```

Comment:
I think the test description needs to change, as an `Annotation` is rendered, technically.